package.json

```diff
@@ -51,7 +51,7 @@
     "test:accessibility": "./script/run-nightwatch.sh --env accessibility",
     "test:accessibility:docker": "docker-compose -p accessibility up -d && docker-compose -p accessibility run --rm --entrypoint=npm -e BABEL_ENV=test -e BUILDTYPE=vagovprod vets-website --no-color run nightwatch:docker -- --env=accessibility && docker-compose -p accessibility stop",
     "test:best-practice": "./script/run-nightwatch.sh --env bestpractice",
-    "test:contract": "./script/run-pact-tests.sh",
+    "test:contract": "BUILDTYPE=localhost npm run test:unit 'src/**/tests/**/*.pact.spec.js'",
```

Comment:
Works just fine using the existing unit test script. Removed the more specific Pact test one to reduce clutter.